components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -1315,6 +1315,8 @@
 
     public static final String GET_API_ID_SQL =
             "SELECT API.API_ID FROM AM_API API WHERE API.API_PROVIDER = ? AND API.API_NAME = ? AND API.API_VERSION = ? ";
+    public static final String GET_API_ID_SQL_NEW =
```

Comment:
lets rename this to GET_API_ID_SQL_BY_UUID or use GET_API_ID_SQL

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10487/commits/f7cce7cd884bf9fc58efecaaad4359ffe282bc84